compiler/tccgen.nim

```diff
@@ -8,12 +8,20 @@
 #
 
 import
-  os, strutils, options, msgs, tinyc
+  os, strutils, options, msgs, tinyc, lineinfos, sequtils
 
-{.compile: "../tinyc/libtcc.c".}
+const tinyPrefix = "dist/nim-tinyc-archive".unixToNativePath
+const nimRoot = currentSourcePath.parentDir.parentDir
+const tinycRoot = nimRoot / tinyPrefix
+when not dirExists(tinycRoot):
+  static: doAssert false, $(tinycRoot, "requires: ./koch installdeps tinyc")
```

Comment:
https://nim-lang.github.io/Nim/manual.html#pragmas-fatal-pragma ?

Reply:
what's the advantage concretely ?